master_server/master_server.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/python
+
+"""This script works on the asssumption of a certain directory structure.
+
+
+     It then acts the master server and load balances the incoming
+     requests.
+     A database is created to store the incoming requests from user.
+"""
+import os
+import requests
+from flask import Flask
+from flask import request
+from flask import render_template
+from flask_sqlalchemy import SQLAlchemy
+import Request_pb2
+
+class MyFlaskApp(Flask):
+  """
```

Comment:
Start comment from here.

Reply:
In general for all comments, in this and other files, please start from the first line and use a single line for single line comments. Thanks!

Reply:
I have made the changes as suggested.